public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -215,5 +217,8 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
         this.genderGraph = 'BS';
       }
   }
+  public convertToNum(s) {
```

Comment:
nit: I would probably just inline this, since it's pretty common Javascript and this method name isn't any shorter than typing `Number`

Reply:
The html somehow does not recognize js function so i put it this way.

Reply:
the inner html does not recognize i mean. Anyways changing the highlight part changes this too.